components/inputs/input-date-time-range-to.js

```diff
@@ -7,6 +7,15 @@ import { LocalizeCoreElement } from '../../helpers/localize-core-element.js';
 import ResizeObserver from 'resize-observer-polyfill/dist/ResizeObserver.es.js';
 import { SkeletonMixin } from '../skeleton/skeleton-mixin.js';
 
+function debounce(func, delay) {
+	let timer = 0;
+	return function() {
+		clearTimeout(timer);
+		// eslint-disable-next-line no-invalid-this
+		timer = setTimeout(() => func.apply(this), delay);
```

Comment:
Would this also work, or does that give it access to the wrong `this`?
```suggestion
		timer = setTimeout(func, delay);
```

Alternately maybe the above combined with `new ResiveObserver(debounce(async() { ... }.bind(this), 5))` down below...